src/vivarium_public_health/metrics/utilities.py

```diff
@@ -177,7 +184,10 @@ def get_age_sex_filter_and_iterables(config: dict, age_bins: pd.DataFrame) -> (
     age_sex_filter = QueryString("")
     if config['by_age']:
         ages = age_bins.set_index('age_group_name').iterrows()
-        age_sex_filter += '{age_group_start} <= age and age < {age_group_end}'
+        if in_span:
+            age_sex_filter += '{age_group_start} < age_at_span_end and age_at_span_start < {age_group_end}'
```

Comment:
should this be age_group_start <= age_at_span_end since we have the start of age bins being inclusive? or is the end of the span exclusive?

Reply:
Seconded. Unless it is exclusive I think it should count as in span based on our inclusive lower bound standard. Though our age precision means this will be rare, right ?